src/app/view/perspectives/post-it/util/selection-helper.ts

```diff
@@ -246,7 +246,7 @@ export class SelectionHelper {
   }
 
   private lastRow(): number {
-    return Object.keys(this.selectedPostIt.document.data).length;
+    return Object.entries(this.selectedPostIt.document.data).length;
```

Comment:
Doesn't it return the same result?

Reply:
It does. It's a debugging leftover.. that does the same thing.